content/en/faq/metrics-and-logs/prometheus-application-metrics.md

```diff
@@ -3,7 +3,7 @@ title: Can I use Prometheus to scrape application metrics with Istio?
 weight: 90
 ---
 
-Yes. Istio ships with [configuration for Prometheus]({{< github_file >}}/manifests/istio-telemetry/prometheus/templates/configmap.yaml)
```

Comment:
`{{< github_file >}}` is a template tag and will be replaced by deployment scripts with `https://raw.githubusercontent.com/istio/istio/<branch_name>`

You only need to update
```
{{< github_file >}}/manifests/charts/istio-telemetry/prometheus/templates/configmap.yaml
```